app/src/main/res/layout/covid_layout.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <data>
+
+        <variable
+            name="covid"
+            type="cl.maleb.testcarsales.data.Covid" />
+
+        <variable
+            name="viewModel"
+            type="cl.maleb.testcarsales.ui.covid.CovidViewModel" />
+
+    </data>
+
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+
+        <ProgressBar
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:visibility="@{viewModel.isShowProgress()}" />
+
+        <LinearLayout
+            android:id="@+id/contentLinear"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_centerInParent="true"
+            android:orientation="vertical"
+            android:padding="@dimen/padding_default"
+            android:visibility="@{viewModel.isShowContent()}">
+
+            <TextView
+                android:id="@+id/tvDate"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:text="@{@string/date + ' ' + covid.date}"
+                android:textAlignment="center"
+                android:textSize="24sp" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="@dimen/margin_default"
+                android:orientation="vertical">
+
+                <TextView
+                    android:id="@+id/tvConfirmedCases"
```

Comment:
As you can see, the only difference in these 2 TextViews is the **id** and the **text**. Then you can create a **style** that has all others fields and use it in both Textviews

Reply:
True.